calvin_bringup/launch/sick_lms200.launch

```diff
@@ -4,7 +4,8 @@
     <remap from="scan" to="lms200" />
     <param name="port" value="/dev/scanner" />
     <param name="baud" value="500000" />
-    <param name="resolution" value="1.0" />
+    <param name="angle" value="180" />
+    <param name="resolution" value="0.5" />
```

Comment:
On Wed, Feb 18, 2015 at 02:33:19PM -0800, Jochen Sprickerhof wrote:
> I'm against the 0.5 degree option, as it's done by combining two scans.

I removed the 0.5 resolution option and we'll fix this by adjusting the scanner firmware.